libnavigation-ui/src/main/res/layout-land/instruction_content_layout.xml

```diff
@@ -28,8 +28,8 @@
             android:layout_width="40dp"
             android:layout_height="40dp"
             android:layout_margin="8dp"
-            app:maneuverViewPrimaryColor="?attr/navigationViewBannerManeuverPrimary"
-            app:maneuverViewSecondaryColor="?attr/navigationViewBannerManeuverSecondary"/>
```

Comment:
the main change is to remove all the `"?attr"` usage in the layouts and get those values from customized attributes and then apply those colors programmatically.